lib/util/git/index.ts

```diff
@@ -655,6 +655,15 @@ export async function commitFiles({
     const commitRes = await git.commit(message, [], {
       '--no-verify': null,
     });
+    if (
+      commitRes.summary &&
+      commitRes.summary.changes === 0 &&
+      commitRes.summary.insertions === 0 &&
+      commitRes.summary.deletions === 0
```

Comment:
```suggestion
      commitRes.summary?.changes === 0 &&
      commitRes.summary?.insertions === 0 &&
      commitRes.summary?.deletions === 0
```

Reply:
Will this result in one less TS line of code but 2 more JS evaluations once transpiled?